e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -34,6 +34,12 @@
             "env": {
               "CollectMetrics": {
                 "value": "true"
+              },
+              "experimentalfeatures__enabled": {
+                  "value": "true"
+              },
+              "experimentalfeatures__enableMetrics": {
+                  "value": "true"
```

Comment:
Hmmm... I'm debating if this should be passed down from pipeline or it's ok to do it by default here. 
How likely that one wants to enable `experimentalfeatures__enabled` and `experimentalfeatures__enableMetrics` in EdgeHub in LH pipeline beyond this PR?

Reply:
Ask Philips what he thinks.

Reply:
Bear and I discussed in person, and since the metrics collector will always be part of longhaul and stress deployments, then hardcoding these experimental features is fine.